static/app/components/eventTitleTreeLabel.tsx

```diff
@@ -3,9 +3,10 @@ import styled from '@emotion/styled';
 
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 import space from 'app/styles/space';
+import {TreeLabelPart} from 'app/types';
 
 type Props = {
-  treeLabel: string[];
+  treeLabel: TreeLabelPart[];
```

Comment:
I believe that at this point you will have an array of strings only 
```suggestion
  treeLabel: string[];
```